addons/MountRoulette/MountRoulette.lua

```diff
@@ -43,6 +47,13 @@ for _, mount in pairs(resources.mounts) do
     possible_mounts:append(mount.name:lower())
 end
 
+blacklist = {}
```

Comment:
This should really be a set `S{}`. And so should `settings.blacklist`. In fact, not sure why you don't make `settings.blacklist` a set and use that directly. Does that not work for you?

Reply:
Ahh yes you're right! I think I had some issues with lists and saving them in settings (I think it was saving them as `<0>...</0><1>...</1>`), but since then I swapped to using sets, so I've changed this to just use the set directly as suggested and it works fine. Thanks!